test/pack_test.cpp

```diff
@@ -841,5 +841,476 @@ TEST_F(PackTest, UnPackItem_ear)
 	ComparePackedItems(is, is2);
 }
 
+class NetPackTest : public ::testing::Test {
+public:
+	void SetUp() override
+	{
+		Players.resize(2);
+		MyPlayer = &Players[0];
+
+		PlayerPack testPack { 0, 0, -1, 9, 0, 2, 61, 24, 0, 0, "MP-Warrior", 0, 120, 25, 60, 60, 37, 0, 85670061, 3921, 13568, 13568, 3904, 3904, { 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0 }, 1610612737, { { 3160997530, 798, 52, 5, 40, 40, 0, 0, 0, 0 }, { 1911837482, 286, 152, 3, 0, 0, 0, 0, 0, 0 }, { 1262914017, 286, 151, 3, 0, 0, 0, 0, 0, 0 }, { 3846694361, 286, 155, 3, 0, 0, 0, 0, 0, 0 }, { 3454746195, 2077, 122, 3, 60, 60, 0, 0, 0, 0 }, { 1560055601, 4117, 75, 3, 50, 50, 0, 0, 0, 0 }, { 3097669048, 286, 70, 3, 66, 90, 0, 0, 0, 0 } }, { { 423576018, 16400, 82, 0, 0, 0, 0, 0, 0, 0 }, { 543375803, 16400, 81, 0, 0, 0, 0, 0, 0, 0 }, { 1802859062, 8208, 147, 3, 35, 35, 34, 34, 0, 0 }, { 368922902, 0, 0, 0, 0, 0, 0, 0, 3921, 0 }, { 1424628865, 8208, 111, 1, 0, 0, 0, 0, 0, 0 }, { 1128267486, 1, 25, 0, 0, 0, 0, 0, 0, 0 }, { 885129636, 1, 25, 0, 0, 0, 0, 0, 0, 0 }, { 1954673116, 260, 93, 0, 0, 0, 0, 0, 0, 0 }, { 379449999, 260, 93, 0, 0, 0, 0, 0, 0, 0 }, { 1684106752, 24937, 23, 0, 0, 0, 0, 0, 65, 0 }, { 1924887070, 385, 24, 0, 0, 0, 0, 0, 0, 0 }, { 813588879, 194, 56, 2, 6, 18, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 } }, { -3, -3, -12, -12, 0, 0, 0, 0, -5, -5, -3, -3, -12, -12, 0, 0, 0, 0, 5, -5, 3, -3, 12, -12, 0, 0, 0, 0, 0, 0, 2, 1, 6, 7, 8, 9, 10, 11, 4, 0 }, 12, { { 112017676, 16, 24, 0, 0, 0, 0, 0, 0, 0 }, { 367533949, 16, 24, 0, 0, 0, 0, 0, 0, 0 }, { 778528030, 16, 24, 0, 0, 0, 0, 0, 0, 0 }, { 960294091, 16, 24, 0, 0, 0, 0, 0, 0, 0 }, { 1948311560, 16, 29, 0, 0, 0, 0, 0, 0, 0 }, { 639577687, 16, 29, 0, 0, 0, 0, 0, 0, 0 }, { 103356930, 16, 29, 0, 0, 0, 0, 0, 0, 0 }, { 215810455, 16, 29, 0, 0, 0, 0, 0, 0, 0 } }, 0, 0, 0, 0, 0, 0, 0, 0, 0, { 0, 0 }, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }, 0, 0, 0, 0, { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 } };

```

Comment:
Could we line break this for each struct inside of it. Some editors handle extream long lines poorly.